pydarn/plotting/maps.py

```diff
@@ -479,3 +501,206 @@ def index_legendre(l: int, m: int):
                                    -velocity_fit_vectors[0,
                                                          velocity_chk_zero_inds])
         return velocity, azm_v
+
+
+    @classmethod
+    def plot_heppner_maynard_boundary(cls, mlats: list, 
+                                      mlons: list, line_color: str='black'):
+        # TODO: No evaluation of coordinate system made! May need if in 
+        # plotting to plot in radians/geo ect.
+        """
+        Plots the position of the Heppner-Maynard Boundary
+
+        Parameters
+        ----------
+            ax: object
+                matplotlib axis object
+            mlats: List[float]
+                Magnetic Latitude in degrees
+            mlons: List[float]
+                Magnetic Longitude in radians
+
+        """
+        plt.plot(mlons,mlats, c=line_color, linewidth=1)
+
+
+    @classmethod
+    def calculate_potentials(cls, fit_coefficient: list, lat_min: list,
+                             lat_shift: int=0, lon_shift: int=0, 
+                             fit_order: int=6, **kwargs):
+        # TODO: No evaluation of coordinate system made! May need if in 
+        # plotting to plot in radians/geo ect.
+        '''
+        Calculates potential across a magnetic lat/lon grid for
+        plotting later
+
+        Parameters
+        ----------
+            fit_coefficient: List[float]
+                Value of the coefficient
+            lat_min: List[float]
+                Minimum latitude that will be evaluated
+                Not to be confused with 'lowlat'
+            lat_shift: int
+                Generic shift in latitude from map file
+                default: 0
+            lon_shift: int
+                Generic shift in longitude from map file
+                default: 0
+            fit_order: int
+                order of the fit
+                default: 6
+            lowlat: int
+                Lowest latitude on plot
+                default: 30
+            hemisphere: Enum
+                Describes the hemisphere, North or South
+                default: Hemisphere.North
+ 
+        '''
+        # Deal with required kwargs
+        try:
+            lowlat = kwargs['lowlat']
+        except KeyError:
+            lowlat = 60
+
+        try: 
+            hemisphere = kwargs['hemisphere']
+        except KeyError:
+            hemisphere = Hemisphere.North
+
+        # Lowest latitude to calculate potential to
+        theta_max = np.radians(90-np.abs(lat_min))
+
+        if hemisphere == Hemisphere.North:
+            hemi = 1
+        else:
+            hemi = -1
+
+        # Make a grid of the space the potential is evaluated on
+        # in magnetic coordinates
+        lat_step = 1
+        lon_step = 2
+        num_lats = int((90.0 - lowlat) / lat_step) + 1
+        num_lons = int(360.0 / lon_step) + 1
+        lat_arr = np.array(range(num_lats)) * lat_step + lowlat
+        lat_arr = lat_arr * hemi
```

Comment:
``` suggestion
        lat_arr = lat_arr * hemisphere
```

Reply:
Answer for future Carley who will come searching for this: `hemisphere.value`!